.vscode/setting.json

```diff
@@ -0,0 +1,9 @@
+{
+    "editor.codeActionsOnSave": {
+        "source.fixAll": true
+    },
+    "editor.formatOnSave": true,
+    "javascript.format.enable": false,
+    "material-icon-theme.activeIconPack": "react_redux"
+
+}
```

Comment:
This file should be `settings.json`. It seems you have both `setting.json` and `settings.json`.  

`setting.json` is actually not recognised by vscode, so you'll need to move all the content of `setting` to `settings.json` which already lives in the same directory